examples/jss/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import './global-styles';
+import GlobalStyles from './components/global-styles';
 import App from './components/app';
 
-render(<App />, document.getElementById('example-root'));
+render(<GlobalStyles><App /></GlobalStyles>, document.getElementById('example-root'));
```

Comment:
This is actually possible in jss by using nested selectors, you are able to use global class names, so if you add a jss class to the html element, you can do this:

```js
{
  html: {
    '&': {htmlStyles}
    '& body': {bodyStyles}
  }
}
```